source/IOSResolver/src/IOSResolver.cs

```diff
@@ -1553,19 +1486,6 @@ public enum CocoapodsIntegrationMethod {
         project.ReadFromString(File.ReadAllText(pbxprojPath));
         string target = project.TargetGuidByName(TARGET_NAME);
         project.SetBuildProperty(target, "CLANG_ENABLE_MODULES", "YES");
```

Comment:
This ought to be taken care of by pods integration, but I wasn't sure if it was needed for GTMSessionFetcher (see comment below), so I left it in.